packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -151,6 +152,23 @@ export function usePress(props: PressHookProps): PressResult {
       }
     };
 
+    let triggerPressUp = (originalEvent: EventBase, pointerType: PointerType) => {
+      if (isDisabled) {
+        return;
+      }
+
+      if (onPressUp) {
+        onPressUp({
+          type: 'pressup',
+          pointerType,
+          target: originalEvent.target as HTMLElement,
```

Comment:
spread the original event?

Reply:
That would include a lot more properties than we want.